game/screen.hh

```diff
@@ -140,6 +140,7 @@ private:
 #endif
 };
 
+#if !(_MSC_VER && !__INTEL_COMPILER) //MSVC does not supports redefintions of specialized static member varables
```

Comment:
did this change with VS 2019 16.5+?
see https://stackoverflow.com/questions/32321006/resolving-definitions-of-specialized-static-member-variables-of-templated-classe

Reply:
Item has been moved to another pR (#537 )